server/src/common/completions.ts

```diff
@@ -5,7 +5,7 @@ import {
 	TextEdit,
 	CompletionItem,
 	CompletionItemKind
-} from 'vscode-languageserver/node';
+} from 'vscode-languageserver/browser';
```

Comment:
Should this just pull it from `vscode-languageserver` instead of either the `node` or `browser` piece?